README.md

```diff
@@ -186,15 +186,19 @@ Offical exam objectives you review and understand in order to pass the test.
 
 
 1. Perform behavioural analytics of syscall process and file activities at the host and container level to detect malicious activities
-  - [Old kubernetes.io URL: install falco on k8s 1.17](https://v1-17.docs.kubernetes.io/docs/tasks/debug-application-cluster/falco/)
+  - [Old kubernetes.io URL: install Falco on k8s 1.17](https://v1-17.docs.kubernetes.io/docs/tasks/debug-application-cluster/falco/)
```

Comment:
needs `:triangular_flag_on_post:` after the dash before the square bracket, same should be applied to any external resources that are not kubernetes.io/{docs|blog} or github.com/kubernetes